src/Artsy/Router/Utils/Route.tsx

```diff
@@ -1,79 +1,50 @@
 /**
  * Taken from @taion as per example of how they actually use found-relay and
  * have default setup for each route.
- *
- * TODO:
- * - Decide if we want to change our getDataFrom, such as the one @taion uses.
- * - There's opportunity to unify some code with renderWithLoadProgress.
  */
 
-import { Spinner } from "@artsy/palette"
-import { LoadingClassName } from "Artsy/Relay/renderWithLoadProgress"
+import { RouteSpinner } from "Artsy/Relay/renderWithLoadProgress"
 import BaseRoute from "found/lib/Route"
 import React from "react"
-import styled from "styled-components"
 
-const SpinnerContainer = styled.figure`
-  width: 100%;
-  height: 100px;
-  position: relative;
-`
+type FetchIndicator = "spinner" | "overlay"
 
-export const renderSpinner = () => {
-  return (
-    <SpinnerContainer className={LoadingClassName}>
-      <Spinner />
-    </SpinnerContainer>
-  )
+interface CreateRenderProps {
+  fetchIndicator: FetchIndicator
 }
 
-// When popping a route (navigate back), use the data in the store.
-// function defaultGetDataFrom({ location }) {
-//   return location.action === "POP" ? "STORE_OR_NETWORK" : "STORE_THEN_NETWORK"
-// }
+interface RenderArgProps {
+  Component: React.ComponentType
+  props?: object
+  error?: Error
+}
 
-function createRender({
-  prerender,
-  render,
-  renderFetched,
-  showFetchIndicator = false,
-}) {
-  return (renderArgs: {
-    Component: React.ComponentType
-    props?: object
-    error?: Error
-  }) => {
+function createRender({ fetchIndicator = "spinner" }: CreateRenderProps) {
+  return (renderArgs: RenderArgProps) => {
     const { Component, props, error } = renderArgs
 
     if (error) {
       throw error
     }
 
-    if (prerender) {
-      prerender(renderArgs)
-    }
-
-    if (render) {
-      return render(renderArgs)
-    }
```

Comment:
Were these just never used?

Reply:
Yah, this was copy / paste from taion and were, I think, functions on his route defintion

Reply:
But we are using `render`, so let me revert this one line.